cms/server/admin/static/aws_tp_styles.css

```
@@ -3266,3 +3266,126 @@
+}
+
+.network-group {
+    border: 1px solid var(--tp-border, #e5e7eb);
+    border-radius: 8px;
+    overflow: hidden;
+    background: var(--tp-bg-white, #fff);
+    transition: box-shadow 0.15s ease;
+}
+
+.network-group:hover {
+    box-shadow: 0 1px 4px rgba(0, 0, 0, 0.06);
```

Stylelint (17.2.0)

[error] Expected "0.06" to be "6%" (alpha-value-notation)

(alpha-value-notation)